src/modules/Colorbar/Module.js

```diff
@@ -1,41 +1,17 @@
-module.exports = function Colorbar(options, UI) {
-
-  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
-  var output;
-
-  options.x = options.x || defaults.x;
-  options.y = options.y || defaults.y;
-  options.colormap = options.colormap || defaults.colormap;
-  options.h = options.h || defaults.h;
-
-  var steps = [
-    { 'name': 'gradient', 'options': {} },
-    { 'name': 'colormap', 'options': { colormap: options.colormap } },
-    { 'name': 'crop', 'options': { 'y': 0, 'h': options.h } },
-    { 'name': 'overlay', 'options': { 'x': options.x, 'y': options.y, 'offset': -4 } }
-  ];
-
-  // ui: false prevents internal logs
-  var internalSequencer = ImageSequencer({ inBrowser: false, ui: false });
-
-  function draw(input, callback) {
-
-    var step = this;
-
-    internalSequencer.loadImage(input.src, function onAddImage() {
-      internalSequencer.importJSON(steps);
-      internalSequencer.run(function onCallback(internalOutput) {
-        step.output = { src: internalOutput, format: input.format };
-        callback();
-      });
-    });
-
-  }
-
-  return {
-    options: options,
-    draw: draw,
-    output: output,
-    UI: UI
+module.exports = require('../../util/createMetaModule.js')(
```

Comment:
see how simple this is? we can also in theory make it accept a string instead of a `mapFunction` and just expose all options by some basic rules... like, if there are 2 rotate modules, how are the settings exposed differently for each rotate step?